.scrutinizer.yml

```diff
@@ -0,0 +1,12 @@
+filter:
+    excluded_paths: [tests/*]
+build:
+    tests:
+        override:
+            -
+                command: 'vendor/bin/phpunit --coverage-clover=coverage.clover'
```

Comment:
Can you fetch a test success/failure status from scrutinizer? If not, I would suggest adding a `.travis.yml` file